### ISSUE TITLE
🐛 BUG: auth fails during an app creation via CLI

### ISSUE THREAD

**Issue by AndrewKushnir** [cloudflare/workers-sdk#3270]
### Which Cloudflare product(s) does this pertain to?

C3

### What version of `Wrangler` are you using?

0.0.0-09fb7272

### What operating system are you using?

Mac

### Describe the Bug

Steps to reproduce:

1. Make sure you are not logged in
2. Run `npm create cloudflare@2.0.4`
3. Try to create an application (I've tried Angular, but I guess it'd be the same behavior for other frameworks as well)
4. When prompted "Do you want to deploy your application?", select `Yes`.

The error is:
```
╭ Deploy with Cloudflare Step 3 of 3
│ 
├ Do you want to deploy your application?
│ yes deploying via `npm run deploy`
│
 ⛅️ wrangler 0.0.0-09fb7272
---------------------------
Getting User settings...

✘ [ERROR] Not logged in.
```

After that, when I go into an app folder and run `npm run deploy` from there - an auth link is presented, I can login and deploy the app. I suspect that the command that kicks off auth is invoked from the current folder (thus unable to locate key information) vs being invoked from the app folder.

**Comment by admah** [comment]
@AndrewKushnir I believe this is fixed as of 2.0.8. I just ran through your repro steps using `npm create cloudflare@latest` for the second step and was successful at deploying a new Angular app. Could you retry?

**Comment by AndrewKushnir** [comment]
@admah thanks for the fix 👍
